xlml/apis/test_config.py

```diff
@@ -263,6 +263,41 @@ def _load_compiled_jsonnet(test_name: str) -> Any:
   return test
 
 
+@attrs.define
+class GpuGkeTest(TestConfig[Gpu]):
```

Comment:
Same here - just wondering if we can consolidate with the TpuGkeTest config. cc @will-cromar

Reply:
Will and I are thinking about refactor on those in coming weeks. Personally, I am fine with this part. Glad to hear more suggestions

Reply:
I agree, I'm fine with this part in this PR, but this will get refactored fairly soon. We have a poor separation between the "what" and a "how" of the test. Any test that runs on GPU in docker should be using the same class or at least interface, and the particulars (like joining the commands with a semicolon to hack around `xpk`'s whitespace handling) should be part of the `Task`. (Also we'll rename `Task` :sweat_smile:). Stuff like `num_slices` should be part of the `accelerator`.

No action is required here IMO, unless rolling these two classes is trivial.

Reply:
micro-nit on the name, actually: this is the second GPU test type that runs on GKE (see `JSonnetGpuTest`). @NinaCai would you mind renaming this to `GpuXpkTest` to make it more specific?